PyOpenWorm/channel.py

```diff
@@ -92,16 +95,16 @@ def __init__(self, name=False, **kwargs):
         Models(owner=self)
         Channel.DatatypeProperty('subfamily', owner=self)
         Channel.DatatypeProperty('description', owner=self)
-        Channel.DatatypeProperty('name', self) #channel_name
-        Channel.DatatypeProperty('description',self) #description
-        Channel.DatatypeProperty('gene_name', self) #gene_name
-        Channel.DatatypeProperty('gene_WB_ID', self) #gene_WB_ID
-        Channel.DatatypeProperty('expression_pattern', self) #expression_pattern
-        Channel.DatatypeProperty('proteins', self, multiple=True) #proteins
-        #TODO: assert this in the adapter instead
-        #Channel.DatatypeProperty('description_evidences', self)
-        #TODO: assert this in the adapter instead
-        #Channel.DatatypeProperty('expression_evidences', self)
+        Channel.DatatypeProperty('name', self)
+        Channel.DatatypeProperty('description', self)
+        Channel.DatatypeProperty('gene_name', self)
+        Channel.DatatypeProperty('gene_WB_ID', self)
+        Channel.DatatypeProperty('expression_pattern', self)
+        Channel.DatatypeProperty('proteins', self, multiple=True)
+        # TODO: assert this in the adapter instead
```

Comment:
Was this a ChannelWorm-related TODO?
Added at 38199eca317505473c4f3940d39a917896760585, 27 Jul 2015